dev/source/docs/building-setup-windows-cygwin.rst

```diff
@@ -127,6 +129,19 @@ Set up directories/paths and extra packages in Cygwin
     pip2 install pyserial
     pip2 install pymavlink
 
+Download Ardupilot Source
+=========================
+
+The Ardupilot source files then need to be downloaded to your local Cygwin filesystem by cloning the git repository and updating the submodules. Enter the following commands into the Cygwin terminal to download the source files:
+
+::
+
+    git clone https://github.com/ardupilot/ardupilot.git
+    cd ardupilot
+    git submodule init
```

Comment:
This should be a single line like [the other wikis](http://ardupilot.org/dev/docs/git-clone.html). I can make that edit for you real quick.
git submodule update --init --recursive

Reply:
oh, weird that all the windows wikis have them on different lines like that already